src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,188 @@
+const { t } = useI18n()
+
+interface Form {
+  status: boolean
+  uuid: string
+}
+
+const form = ref<Form>({
+  status: false,
+  uuid: ''
+})
+
+interface ErrorObject {
+  notificationTitle: string
+  notificationDescription: string
+  uuid: string
+}
+const objError: ErrorObject = {
+  notificationTitle: '',
+  notificationDescription: '',
+  uuid: ''
+}
+
+let isError = ref(false)
+let loading = ref(false)
+let saving = ref(false)
+let uuidRef = ref()
+let JustCopied = ref(false)
+
+const error: Ref<ErrorObject> = ref({ ...objError })
+const errorLoadingConfiguration: Ref<ErrorObject> = ref({ ...objError })
+const errorSaving: Ref<ErrorObject> = ref({ ...objError })
+
+onMounted(() => {
+  getConfiguration()
+})
+
+async function getConfiguration() {
+  try {
+    loading.value = true
+    let getDataConfiguration = await ubusCall('ns.netifyd', 'status', {})
+    if (getDataConfiguration && getDataConfiguration.data) {
+      let configuration = getDataConfiguration.data
+      form.value.status = configuration.enabled
+      form.value.uuid = configuration.uuid
+      loading.value = false
+    }
+  } catch (exception: any) {
+    isError.value = true
+    errorLoadingConfiguration.value.notificationTitle = t(
+      'error.cannot_retrieve_netify_informatics_configuration'
+    )
+    errorLoadingConfiguration.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  }
+}
+
+function clearErrors() {
+  error.value = {
+    notificationTitle: '',
+    notificationDescription: '',
+    uuid: ''
+  }
+}
+
+function handleSaveAutomaticToggle() {
+  clearErrors()
+  saving.value = true
+  const status = form.value.status ? 'enable' : 'disable'
+  ubusCall('ns.netifyd', status)
+    .then((response) => {
+      if (response.data && response.data.result && response.data.result === 'success') {
+        setTimeout(() => {
+          notificationsStore.createNotification({
+            title: t('standalone.netify_informatics.netify_informatics_' + status),
+            description: t(
+              'standalone.netify_informatics.netify_informatics_' + status + '_description'
+            ),
+            kind: 'success'
+          })
+        }, 500)
+      }
+    })
+    .catch((exception: AxiosError) => {
+      errorSaving.value.notificationTitle = t('error.cannot_save_configuration')
+      errorSaving.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    })
+    .finally(() => (saving.value = false))
+}
+
+function open_dashboard() {
+  window.open('https://portal.netify.ai/login', '_blank')
+}
+function copy_uuid() {
+  navigator.clipboard.writeText(form.value.uuid)
+  JustCopied.value = true
+  setTimeout(() => {
+    JustCopied.value = false
+  }, 3000)
+}
+</script>
+
+<template>
+  <div>
+    <FormLayout class="max-w-6xl">
+      <template #description>
+        <i18n-t keypath="standalone.netify_informatics.content_description" tag="p" scope="global">
+        </i18n-t>
```

Comment:
Use `t` object in template for every localized string
```suggestion
        <p>{{ t('standalone.netify_informatics.content_description') }}</p
```